src/main/java/de/craften/ui/swingmaterial/fonts/MaterialIcons.java

```diff
@@ -0,0 +1,976 @@
+    public static final char VIEW_QUILT = '\uE8F1';
+    public static final char VIEW_STREAM = '\uE8F2';
+    public static final char VIEW_WEEK = '\uE8F3';
+    public static final char VIGNETTE = '\uE435';
+    public static final char VISIBILITY = '\uE8F4';
+    public static final char VISIBILITY_OFF = '\uE8F5';
+    public static final char VOICE_CHAT = '\uE62E';
+    public static final char VOICEMAIL = '\uE0D9';
+    public static final char VOLUME_DOWN = '\uE04D';
+    public static final char VOLUME_MUTE = '\uE04E';
+    public static final char VOLUME_OFF = '\uE04F';
+    public static final char VOLUME_UP = '\uE050';
+    public static final char VPN_KEY = '\uE0DA';
+    public static final char VPN_LOCK = '\uE62F';
+    public static final char WALLPAPER = '\uE1BC';
+    public static final char WARNING = '\uE002';
+    public static final char WATCH = '\uE334';
+    public static final char WATCH_LATER = '\uE924';
+    public static final char WB_AUTO = '\uE42C';
+    public static final char WB_CLOUDY = '\uE42D';
+    public static final char WB_INCANDESCENT = '\uE42E';
+    public static final char WB_IRIDESCENT = '\uE436';
+    public static final char WB_SUNNY = '\uE430';
+    public static final char WC = '\uE63D';
+    public static final char WEB = '\uE051';
+    public static final char WEB_ASSET = '\uE069';
+    public static final char WEEKEND = '\uE16B';
+    public static final char WHATSHOT = '\uE80E';
+    public static final char WIDGETS = '\uE1BD';
+    public static final char WIFI = '\uE63E';
+    public static final char WIFI_LOCK = '\uE1E1';
+    public static final char WIFI_TETHERING = '\uE1E2';
+    public static final char WORK = '\uE8F9';
+    public static final char WRAP_TEXT = '\uE25B';
+    public static final char YOUTUBE_SEARCHED_FOR = '\uE8FA';
+    public static final char ZOOM_IN = '\uE8FF';
+    public static final char ZOOM_OUT = '\uE900';
+    public static final char ZOOM_OUT_MAP = '\uE56B';
+    
+    public static final Font ICON_FONT = loadFont("MaterialIcons-Regular.ttf");
+    
+    private static Font loadFont(String resourceName) {
+        try (InputStream inputStream = MaterialIcons.class.getResourceAsStream("/fonts/" + resourceName)) {
+            return Font.createFont(Font.TRUETYPE_FONT, inputStream);
+        } catch (IOException | FontFormatException e) {
+            throw new RuntimeException("Could not load " + resourceName, e);
+        }
+        //JDK 6 compatible
+//        try {
+//            Font font;
+//            InputStream inputStream = MaterialIcons.class.getResourceAsStream("/fonts/" + resourceName);
+//            font = Font.createFont(Font.TRUETYPE_FONT, inputStream);
+//            inputStream.close();
+//            return font;
+//        } catch (Exception e) {
+//            throw new RuntimeException("Could not load " + resourceName, e);
+//        }
+    }
+    
+    /*public static void main(String[] args) throws Exception {
```

Comment:
Yeah, it was part of my testing in Windows2K, as there isn't try-with-resources in Java 6.